### ISSUE TITLE
synchronize in pool.py

### ISSUE THREAD

**Issue by ghqing0310** [rusty1s/pyg_autoscale#9]
Hello, GNNAutoScale is an interesting and solid work, and thanks for your hard work.
However, when running "synchronize = torch.ops.torch_geometric_autoscale.synchronize", it reports an error. 
How to deal with this problem?
![image](https://user-images.githubusercontent.com/22017585/141077736-ea1716e0-6df7-432c-9431-e7d91116ec9a.png)

**Comment by rusty1s** [comment]
Did you run `python setup.py install` before executing the scripts?

**Comment by ghqing0310** [comment]
> Did you run `python setup.py install` before executing the scripts?

Yes, and when running small_benchmark/main.py, it shows that a circular import happens. 
![image](https://user-images.githubusercontent.com/22017585/141080727-fa383dc0-d58d-481c-a090-f98c5602465c.png)

**Comment by rusty1s** [comment]
Can you try to run the script not within the root folder, but within `small_benchmark`?
```
cd small_benchmark
python main.py
```

**Comment by ghqing0310** [comment]
> Can you try to run the script not within the root folder, but within `small_benchmark`?
> 
> ```
> cd small_benchmark
> python main.py
> ```

Yep that's the way I run it.
![image](https://user-images.githubusercontent.com/22017585/141081764-9702e7ef-204f-4f7b-aaba-bfebcbeed997.png)

**Comment by rusty1s** [comment]
This is weird. What's the installation log of
```
python setup.py clean && python setup.py develop
```
